src/cron/bundle.rs

```diff
@@ -48,11 +49,11 @@ pub async fn validate_bundler<Context>(
     bundler: Bundler,
 ) -> Result<(), ValidatorCronError>
 where
-    Context: queries::RequestContext,
+    Context: queries::RequestContext + arweave::ArweaveContext,
 {
     let arweave = Arweave::new(80, String::from("arweave.net"), String::from("http"));
     let txs_req = arweave
-        .get_latest_transactions(&bundler.address, Some(50), None)
+        .get_latest_transactions::<Context>(&bundler.address, Some(50), None, ctx)
```

Comment:
There should be no need to have `::<Context>` in the call. It should be inferred through args.